specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/examples/SourceControls_CreateOrUpdate.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "subscriptionId": "651f8027-33e8-4ec4-97b4-f6e9f3dc8744",
+    "resourceGroupName": "workerapps-rg-xj",
+    "containerAppName": "testcanadacentral",
+    "name": "current",
+    "api-version": "2022-01-01-preview",
+    "sourceControlEnvelope": {
+      "properties": {
+        "repoUrl": "https://github.com/xwang971/ghatest",
+        "branch": "master",
+        "githubActionConfiguration": {
+          "registryInfo": {
+            "registryUrl": "xwang971reg.azurecr.io",
+            "registryUserName": "xwang971reg",
+            "registryPassword": "<registrypassword>"
+          },
+          "azureCredentials": {
+            "clientId": "<clientid>",
+            "clientSecret": "<clientsecret>",
+            "tenantId": "<tenantid>"
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/651f8027-33e8-4ec4-97b4-f6e9f3dc8744/resourceGroups/workerapps-rg-xj/providers/Microsoft.Web/containerApps/myapp/sourcecontrols/current",
+        "name": "current",
+        "type": "Microsoft.Web/containerapps/sourcecontrols",
```

Comment:
The resource name is Microsoft.Web, is this correct?

Reply:
@takekazuomi - thanks for pointing this out. We'll fix asap.